src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/BackupConfigurationIT.kt

```diff
@@ -0,0 +1,86 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.browser.DockerisedChrome
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.measure.output.CollectionActionMetricOutput
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.page.isElementPresent
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.openqa.selenium.By
+import java.time.Clock
+import java.util.*
+
+class BackupConfigurationIT {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    /**
+     * During the test, you can connect to WebDriver by the VNC viewer.
```

Comment:
❤️

Reply:
This is so good!
PS. the [VNC password is `secret`](https://github.com/testcontainers/testcontainers-java/blob/06ea95be1e288622aedb3a35ea585575654e5a9a/core/src/main/java/org/testcontainers/containers/VncRecordingContainer.java#L33). Open-source FTW :)